packages/lib-classifier/src/store/feedback/helpers/generate-rules.js

```diff
@@ -23,8 +23,13 @@ function generateRules (subject, workflow) {
 
   _.forEach(workflowRules, (rules, taskId) => {
     const taskRules = _.reduce(rules, (result, workflowRule) => {
-      const matchingSubjectRule = _.find(subjectRules, subjectRule =>
-        subjectRule.id.toString() === workflowRule.id.toString())
+      const matchingSubjectRule = _.find(subjectRules, (subjectRule) => {
+        if (subjectRule.id && workflowRule.id) {
+          return subjectRule.id.toString() === workflowRule.id.toString()
+        }
+
+        return null
```

Comment:
I think the callback for `.find` should return a boolean, if it’s the same as the native JS `Array.find` method. So maybe return `false` rather than relying on `null` being falsy.

Reply:
It doesn't behave the same as the native `find`: https://lodash.com/docs/2.4.2#find

Reply:
Wait, I had to read that a couple of times and I am wrong. Ignore me.